tests/monitoring/monitoring.go

```diff
@@ -446,6 +446,37 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, decorators
 
 			verifyAlertExist("KubeVirtVMStuckInErrorState")
 		})
+
+		It("should expose VM CPU metrics", func() {
+			vm := newVirtualMachine()
+			createVirtualMachine(vm)
+
+			metrics := []string{
+				"kubevirt_vmi_cpu_system_usage_seconds",
+				"kubevirt_vmi_cpu_usage_seconds",
+				"kubevirt_vmi_cpu_user_usage_seconds",
+			}
+
+			for _, metric := range metrics {
+				Eventually(func() error {
+					v, err := getMetricValueWithLabels(virtClient, metric, nil)
+					if err != nil {
+						return err
+					}
+
+					i, err := strconv.Atoi(v)
+					if err != nil {
+						return err
+					}
+
+					if i < 0 {
```

Comment:
@machadovilaca I think for the same of clarity you can use `Should(BeNumerically(">", 0))` here

Reply:
updated